reduce search button padding to match search input height

shpiyu opened the pull request:
The search button's height is little more than the search input, so reduced the padding of the button to match the hieghts. 

Before:
<img width="1512" alt="Screenshot 2024-11-03 at 10 57 18 AM" src="https://github.com/user-attachments/assets/c93e0f92-42b7-4216-a765-1ac3ee278d55">

After: 
<img width="1507" alt="Screenshot 2024-11-03 at 11 00 09 AM" src="https://github.com/user-attachments/assets/34c3c60b-333d-4b0d-8c6b-74187189b8bb">

<img width="962" alt="Screenshot 2024-11-03 at 10 59 41 AM" src="https://github.com/user-attachments/assets/7484aabf-d850-42af-bd13-6bec6b3cb69f">

knadh commented:
This breaks in Firefox. I think we should remove the padding altogether and use flex to align it.

shpiyu commented:
@knadh flexbox works, although the outline was still causing difference in how the input box is rendered in chrome vs firefox.

In firefox:
<img width="1280" alt="Screenshot 2024-11-23 at 6 12 17 PM" src="https://github.com/user-attachments/assets/b1b4af52-d8ca-4df0-b0e7-ee268b1423b5">

In chrome:
<img width="1443" alt="Screenshot 2024-11-23 at 7 51 35 PM" src="https://github.com/user-attachments/assets/13b2420f-251e-42b3-bd36-5c478c379c61">

So, I removed the outline and added an accent color that is applied to the border when the input is focused. Now it looks consistent in both the browsers.
<img width="1443" alt="Screenshot 2024-11-23 at 8 11 23 PM" src="https://github.com/user-attachments/assets/38c7dbca-3b10-42bb-a231-85023eea43cd">
<img width="1443" alt="Screenshot 2024-11-23 at 8 12 56 PM" src="https://github.com/user-attachments/assets/8ba31c8d-0ac9-4a28-b197-70a5781d6d23">
